simple_history/admin.py

```diff
@@ -97,45 +162,108 @@ def history_view(self, request, object_id, extra_context=None):
             request, self.object_history_template, context, **extra_kwargs
         )
 
-    def response_change(self, request, obj):
-        if "_change_history" in request.POST and SIMPLE_HISTORY_EDIT:
-            verbose_name = obj._meta.verbose_name
+    def get_object(self, request, object_id, object_history=None, **kwargs):
+        """Returns the model instance or None
+
+        If None, attempts to get the instance from history.
+        """
+        obj = super(SimpleHistoryAdmin, self).get_object(request, object_id, **kwargs)
+        if not obj and object_history:
+            try:
+                obj = object_history.latest("history_date").instance
+            except ObjectDoesNotExist:
+                pass
+        return obj
+
+    def get_object_history(self, object_id):
+        """Returns a queryset of historical instances.
+        """
+        queryset = self.history_manager.filter(
+            **{self.model._meta.pk.attname: object_id}
+        )
+        if not isinstance(self.history_manager.model.history_user, property):
+            queryset = queryset.select_related("history_user")
+        return self.fetch_history_list_display_callables(queryset)
+
+    @property
+    def history_manager(self):
```

Comment:
We have a util that already does this. but fine with this for now so we can get this in